install/resources/upgrade34.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * MyBB 1.8
+ * Copyright 2014 MyBB Group, All Rights Reserved
+ *
+ * Website: http://www.mybb.com
+ * License: http://www.mybb.com/about/license
+ *
+ */
+
+/**
+ * Upgrade Script: 1.8.5
+ */
+
+$upgrade_detail = array(
+	"revert_all_templates" => 0,
+	"revert_all_themes" => 0,
+	"revert_all_settings" => 0
+);
+
+@set_time_limit(0);
+
+function upgrade34_dbchanges()
+{
+	global $db, $output;
+
```

Comment:
You need to call `print_headers` too. Otherwise the update is broken.

```php
	$output->print_header("Updating Database");
	echo "<p>Performing necessary upgrade queries...</p>";
	flush();
```